custom_components/smartthinq_sensors/sensor.py

```diff
@@ -297,6 +312,84 @@
     DeviceType.WASHER,
 ]
 
+RANGE_SENSORS = {
+    DEFAULT_SENSOR: {
+        ATTR_MEASUREMENT_NAME: "Default",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FN: lambda x: x._power_state,
+        ATTR_ENABLED: True,
+    },
+    FEAT_COOKTOP_STATE: {
+        ATTR_MEASUREMENT_NAME: "Cooktop",
+        ATTR_ICON: DEFAULT_ICON,
+        ATTR_VALUE_FN: lambda x: x._cooktop_state,
+        ATTR_ENABLED: True,
```

Comment:
We really want all sensors enabled or there are someone that we can leave initially disabled?

Reply:
That's a good question. The API has individual fields for all five stovetop burners. However, I have noticed that my device reports status only in the first stovetop burner field. If any of the burners are on, the first field indicates the aggregated status.
What I don't know is whether this behavior is true for all stovetops or only for mine. My stovetop is gas, but maybe some electric stovetops report status individually. It's also possible that some future firmware upgrade enables status for all individual burners.
And, unfortunately, there is no way in the API to detect whether status is reported individually or aggregated.
What do you think?

Reply:
Ok, I disabled some sensors for now. This is what makes sense for the range I own. We can enable some sensors as we come across devices that support multiple burners.

Reply:
This is just initial sensor state, anyone can enable sensors going in the device page. The scope is to not have tons of sensors automatically enabled when device is detected the first time, but just the most important.